app/assets/stylesheets/themes/home.scss

```diff
@@ -31,14 +33,14 @@ h1, h2, h3, h4, h5, h6 {
       text-decoration: none;
     }
     &.red {
-      background: #98002e no-repeat center;
+      background: $cu-red no-repeat center;
       color: #FFFFFF;
       &:hover {
-        background-color: #ba183e;
+        background-color: #C8003F;
```

Comment:
Can we use `$red-button-hover` on this line and `$red-button-active` on line 43?